gulpfile.ts

```diff
@@ -0,0 +1,38 @@
+import del = require("del");
+import * as gulp from "gulp";
+const httpServer = require("http-server");
```

Comment:
This should be `import`, not `const`. If you don't have types available for it, either rely on the js we find, or declare a module shorthand for it.